ui/apps/dashboard/src/components/BillingBanner/parse.tsx

```diff
@@ -71,20 +71,6 @@ export function parseEntitlementUsage(data: EntitlementUsageQuery['account']['en
     }
   }
 
-  if (concurrency.usage >= 12) {
-    issues.add(
-      'concurrency',
-      <div className="flex items-center">
-        Account concurrency limit reached in {concurrency.usage} of the past 24 hours
-        <BillingBannerTooltip>
-          Reaching the concurrency limit adds delays between steps, making function runs take longer
-          to complete.
-        </BillingBannerTooltip>
-      </div>,
-      IssueSeverity.softLimitReached
-    );
-  }
-
```

Comment:
Can we disable it instead of deleting? We may want to bring it back when we fix the underlying metric